src/index.ts

```diff
@@ -34,6 +34,7 @@ import {entity} from './entity';
 import {Query} from './query';
 import {DatastoreRequest} from './request';
 import {Transaction} from './transaction';
+import {google} from '../proto/datastore';
```

Comment:
I'm a little surprised you need this, since it doesn't look like it's used anywhere.

Reply:
Leftover from some testing, sorry about that!